full_cluster_replacement_test.py

```diff
@@ -0,0 +1,64 @@
+import logging
+from longevity_test import LongevityTest
+from sdcm import prometheus
+from sdcm.sct_events.system import InfoEvent
+from sdcm.utils import loader_utils
+
+LOGGER = logging.getLogger(__name__)
+
+
+class FullClusterReplacement(LongevityTest, loader_utils.LoaderUtilsMixin):
+    """These tests should use multi data center (multi-DC) and multi Availability Zone (multi-AZ)
```

Comment:
you didn't set multi-AZ

Reply:
From the plan sounds like the answer is yes

Reply:
added